src/main/java/games/strategy/triplea/ai/proAI/ProPurchaseAI.java

```diff
@@ -121,6 +122,100 @@ int repair(int PUsRemaining, final IPurchaseDelegate purchaseDelegate, final Gam
     return PUsRemaining;
   }
 
+  /**
+   * Default settings for bidding:
+   * 1) Limit one bid unit in a territory or sea zone (until set in all territories then 2, etc)
+   * 2) The nation placing a unit in a territory or sea zone must have started with a unit in said territory or sea zone
+   * prior to placing the bid
+   */
```

Comment:
"First sentence of Javadoc is incomplete (period is missing) or not present."

Adding a period to the end of items (1) and (2) gets rid of the warning.